src/main/java/org/embl/mobie/io/ome/zarr/util/OmeZArrayAttributes.java

```diff
@@ -15,7 +15,7 @@
     private final String dimensionSeparator;
 
     public OmeZArrayAttributes(int zarr_format, long[] shape, int[] chunks, DType dtype, ZarrCompressor compressor,
-                               String fill_value, char order, Collection<Filter> filters, String dimensionSeparator) {
+                               Integer fill_value, char order, Collection<Filter> filters, String dimensionSeparator) {
```

Comment:
Representing this as integer is not alsways correct. It could also be float in some cases, depending on the datatype of the zarr dataset.

Reply:
Ok, thank you! I'll fix it.